distributed/dashboard/components/scheduler.py

```diff
@@ -1253,7 +1253,7 @@ def convert(self, msgs):
             "color": color,
             "duration": total_duration,
             "radius": radius,
-            "cost_factor": min(10, self.steal.cost_multipliers[level]),
+            "cost_factor": self.steal.cost_multipliers[level],
```

Comment:
I was wondering why do we need to keep computing what "cost_factor" is even though we are not using to update the plot. Same would apply to `self.steal`